packages/react/src/components/InAppMessaging/CloseIconButton/CloseIconButton.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+
+import { Link } from '../../../primitives';
+import { IconClose } from '../../../primitives/Icon/internal';
+
+import { CloseIconButtonProps } from './types';
+
+export function CloseIconButton({
+  className,
+  onClick,
+  style,
+  ...rest
+}: CloseIconButtonProps): JSX.Element {
+  return (
+    <Link className={className} onClick={onClick} style={style} {...rest}>
```

Comment:
Can we use a Button primitive here? Additionally, can we:

- add a string prop `dismissButtonLabel`
- add it as an aria-label attribute to this Button
- maybe default the prop to 'Dismiss message'? 

This way the button will have an accessible label. Our Alert primitive uses [a similar pattern](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/src/primitives/Alert/Alert.tsx#L19-L60)